### PR TITLE
Add new InfluxDB plan: small. 1vm/8gb mem/50gb storage

### PR DESCRIPTION
**Pull request by AP-Hunt**
What
----
Creates a larger InfluxDB so that:
1. We can bill tenants who are already on it
2. We can allow tenants to upgrade when they need i

How to review
-------------
Run it down your pipeline and try provision one

---

🚨⚠️ Please do not merge this pull request via the GitHub UI ⚠️🚨

### CODE REVIEW

File: config/service-brokers/aiven/config.json [hunk 1]
```diff
@@ -266,7 +266,7 @@
         },
         "plans": [
           {
-            "id": "f636ed93-3354-4173-b8bd-031f54866528",
+            "id": "f604ab6f-624a-4a7d-9849-4335e55408c4",
```
Comment (paroxp) [inline config/service-brokers/aiven/config.json:269]: I don't think, that's what you want here.

**Review by paroxp — CHANGES_REQUESTED** [review]

File: config/service-brokers/aiven/config.json (hunk 1 above)
Comment (AP-Hunt) [reply config/service-brokers/aiven/config.json:269]: Oh, weird. I don't remember doing that. Guess I wasn't paying attention. Really good catch!

File: config/service-brokers/aiven/config.json [hunk 2]
```diff
@@ -284,6 +284,26 @@
                 "version": "1"
               }
             }
+          },
+          {
+            "id": "f604ab6f-624a-4a7d-9849-4335e55408c4",
+            "name": "small-1.x",
+            "aiven_plan": "startup-8",
+            "description": "NOT Highly Available, 1 dedicated VM, 2 CPU per VM, 8GB RAM per VM, 50GB disk space. Free for trial orgs. Costs for billable orgs.",
+            "free": true,
```
Comment (paroxp) [inline config/service-brokers/aiven/config.json:293]:
```suggestion
            "description": "NOT Highly Available, 1 dedicated VM, 2 CPU per VM, 8GB RAM per VM, 50GB disk space.",
            "free": false,
```

Is this a desired state of this plan?

Comment (AP-Hunt) [reply config/service-brokers/aiven/config.json:293]: Yes, I think it is. Good catch, again.

**Review by paroxp — APPROVED** [review]

**Pull request merged**